apps/src/applab/commands.js

```diff
@@ -431,45 +433,29 @@ applabCommands.getDirection = function(opts) {
   return Applab.turtle.heading;
 };
 
+// whether or not Turtle pen is down, executing command will result in drawing a 'dot'
```

Comment:
super nit: capitalise start of sentence and end with a period, just to make it firmly readable :)